### DISCUSSION TITLE
disabling rounded corners?

### DISCUSSION THREAD

**Discussion by funn-hue** [Maplespe/DWMBlurGlass#197] [Q&A] [answered]
so when using a theme with frames (square) it still looks like the app is adding rounded corners to the blur effect, can it be disabled ? 
![image](https://github.com/Maplespe/DWMBlurGlass/assets/152728509/de117f46-81f4-447a-8306-593d0a0c9f3d)

EDIT: actually that looks like a missing 1 pixel on each corner

**Comment by Maplespe** [comment, marked as answer]
View #99.

**Reply by funn-hue** [reply]
works, thanks
